packages/boot/test/bootstrapTests/orchestration.test.ts

```diff
@@ -24,7 +24,10 @@ test.serial('stakeBld', async t => {
     evalProposal,
     refreshAgoricNamesRemotes,
   } = t.context;
-  // TODO move into a vm-config for u15
+  // TODO move into a vm-config for future agoric-upgrade
```

Comment:
thx for the ping.

The comment is to not run a proposal in test that would already run (either in bootstrap or a previous upgrade).

I see this PR is putting the eval into the upgrade handling: https://github.com/Agoric/agoric-sdk/pull/8624/files#diff-0de0040f0a2dd4f3a120b1db6cce75755e3cd78dc89a268f341b513f0f97aef4R940

So it's already "deployed" and shouldn't be in the test. This test uses the default vm-config, `decentral-main-vaults-config.json`. So just add `init-transfer` there and remove it from here.

And `init-localchain.js` is already in the upgrade handler so it should also move.

Then the comment is obsolete. It could be removed but I'd replace it with something like,
```
Eval what is not yet on chain. If something here gets on chain, move its script into a vm-config so that it's not part of the test spec.
```